.github/workflows/validate-crd.yaml

```diff
@@ -0,0 +1,39 @@
+name: Check CRD Version Update
+
+on:
+  pull_request:
+    paths:
+      - 'pkg/k8s/apis/cilium.io/client/crds/v1alpha1/*.yaml'
+
+jobs:
+  check-version:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        run: echo "changed_files=$(git diff --name-only -r HEAD^1 HEAD | xargs)" >> $GITHUB_ENV
```

Comment:
Since it looks only at the last commit, it won't work for PRs with multiple commits where the last one isn't modifying CRDs, right?

Reply:
Your right . Just confirmed about it . Any suggestions how to proceed further?

Reply:
yes for our case I think since we squash It wun't cause the issues . I'm open to your ideas on this

Reply:
There are definitely ways around this, but it's probably easiest to just use this action for it: https://github.com/marketplace/actions/get-all-changed-files

Reply:
sure , I'll try them

Reply:
thank you @willfindlay . I have updated